### PR TITLE
Add a Jacobian arm-jogging tutorial

### PR DESCRIPTION
**Pull request by AndyZe**
This is a tutorial for the MoveIt! jogging PR (https://github.com/ros-planning/moveit/pull/1244).

I've forked Prof. Nishida's Motoman repository to make a Gazebo example. I hope that's OK, @Yuji-Ninomiya @AtsukiYokota @NishidaTakeshiLab? If not, I can probably find a different example.

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

### CODE REVIEW

**Comment by AtsukiYokota** [comment]
Hi @AndyZe !
If you care about permission to use, it is okay!

**Comment by AndyZe** [comment]
Thank you!

**Comment by AndyZe** [comment]
If there is still any question about linking to the Motoman repo, I got this email from Prof. Nishida. Awesome!

> There is no problem with the license. Please use freely.

> When you have completed developing the tutorial, please let us know.

> I would like to publicly advertise your tutorials.

File: doc/arm_jogging/arm_jogging_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	    y: 0.0
+	    z: 0.0"
+
+Settings
+--------
+User-configurable settings of the jog node are well-documented in ``config/sia5_simulated_config.yaml``.
+
+Configuring Control Devices (Gamepads, joysticks, etc)
+------------------------------------------------------
+The ``jog_arm/config`` folder contains two examples of converting SpaceNavigator 3D mouse commands to jog commands. ``spacenav_teleop_tools.launch`` loads a config file then publishes commands to the jogger on the ``spacenav/joy topic``. It is easy to create your own config file for a particular joystick or gamepad. We welcome pull requests of config files for new controllers.
```
Comment (davetcoleman) [inline doc/arm_jogging/arm_jogging_tutorial.rst]: can you hyperlink SpaceNavigator 3D to Amazon?

File: doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst [hunk 2]
```diff
@@ -12,6 +12,8 @@ If you haven't already done so, make sure you've completed the steps in `Getting
 
 Running the Code
 ----------------
+This jogging should only be used for simulated robots because some IK solvers can cause a joint jump. To jog a real robot, use the Jacobian-based `Real-Time Arm Jogging <../arm_jogging/arm_jogging_tutorial.html>`_ instead.
```
Comment (davetcoleman) [inline doc/joystick_control_teleoperation/joystick_control_teleoperation_tutorial.rst]: this approach doesn't use IK actually - it uses RRT Connect etc to plan for the goal set by the joystick. I don't think this sentence is fair for this approach, there's just different beasts

**Comment by AndyZe** [comment]
Closing this PR to make a Melodic version.

**Pull request closed without merging**